src/OpenLoco/Vehicles/VehicleHead.cpp

```diff
@@ -42,6 +43,8 @@ namespace OpenLoco::Vehicles
     static loco_global<int16_t[128], 0x00503B6A> factorXY503B6A;
     static constexpr uint16_t trainOneWaySignalTimeout = 1920;
     static constexpr uint16_t trainTwoWaySignalTimeout = 640;
+    static constexpr uint16_t busSignalTimeout = 960;
+    static constexpr uint16_t tramSignalTimeout = 2880;
```

Comment:
Do these refer to railroad crossings, by any chance? Their barriers are a kind of signal as well in reality, of course, but in the context of the game it's a bit confusing/overloaded, I think.

Reply:
I still think some clarification here would be nice.